SecondSemester/Calculator/CalculatorForm/Program.cs

```diff
@@ -0,0 +1,17 @@
+namespace CalculatorUI
+{
```

Comment:
```suggestion
namespace CalculatorUI;
```
В моде уже file-scoped неймспейсы :)
в других файлах аналогично